src/content/blog/2025-05-05-celebrating-ten-years-of-discover.mdx

```diff
@@ -63,6 +63,6 @@ We are proud to provide aerial imagery services for these incredible agencies, a
 
 The list could go on and on!
 
-The wide variety of use cases for Discover is truly what's worth celebrating. This imagery wouldn't be worth much if it just sat around collecting dust. But because of the amazing people we've had the joy of working with, these aerial photos become more than just pixels on a screen. Over the last ten years, we have had the opportunity to see these products transform lives and bolster data-driven decision making in our state. From the background processes that underpin everyday life, to the emergency management practices made possible through well-maintained data, Discover is just one way we “encourage and facilitate the effective use of geospatial information and technology for Utah.”
+The wide variety of use cases for Discover is truly what's worth celebrating. These services wouldn't be worth much if they just sat around collecting dust. But because of the amazing people we've had the joy of working with, those service become more than just pixels on a screen. Over the last ten years, we have had the opportunity to see these products transform lives and bolster data-driven decision making in our state. From the background processes that underpin everyday life, to the emergency management practices made possible through well-maintained data, Discover is just one way we “encourage and facilitate the effective use of geospatial information and technology for Utah.”
 
 You can sign up for Discover and learn more about what we provide on our website. For further questions or inquiries, please reach out to <Contacts inline subject={frontmatter.title} contactKey='elevation' />.
```

Comment:
this would benefit from a link to the discover sign up page.

Reply:
done